bundle-workflow/python/build_workflow/builder.py

```diff
@@ -24,8 +24,9 @@ def __init__(self, component_name, git_repo, component_scripts_path, default_bui
         self.component_scripts_path = component_scripts_path
         self.default_build_script = default_build_script
         self.build_recorder = build_recorder
+        self.output_path = 'artifacts'
```

Comment:
Make this a constant declared at the class level:

```
class Builder:
  OUTPUT_PATH = 'artifacts'
```

Reply:
I think this actually should become input as a fully qualified path, so you can redirect things and not always create a hard-coded artifacts dir from where you run things. I didn't want to pollute this PR with that, though.